### PR TITLE
Bump alpine base image to 3.7.4-alpine3.10

### PR DESCRIPTION
**Pull request by TheForcer**
Upgrade to a newer alpine base image version

### CODE REVIEW

**Comment by TheForcer** [comment]
We should also discuss a solution for automatic Docker image updates. Maybe GitHub actions can do that? What do you think @konstantingoretzki ?

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.7.0-alpine3.8
+FROM python:3.7.4-alpine3.10
 
 LABEL authors="Konstantin Goretzki, Felix Alexa"
 LABEL version="v1.0"
```
Comment (konstantingoretzki) [inline Dockerfile]: I guess we should also upgrade the version number to v1.1.

**Comment by konstantingoretzki** [comment]
Hey, thanks, great idea. Let's talk later about the new GitHub feature.

Regarding the issue: Guess we should also upgrade the version label?

File: Dockerfile (hunk 1 above)
Comment (TheForcer) [inline Dockerfile]: 1.1 here?

Comment (konstantingoretzki) [reply Dockerfile]: Exactly.

**Comment by konstantingoretzki** [comment]
Thanks, will merge.

**Pull request merged**